src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1210,6 +1211,32 @@ def mocked_connection_get(*args, **kwargs):
         outerr = capsys.readouterr()
         assert not outerr.err
 
+    def test_tool_init(self):
```

Comment:
Can we add some invalid test case? like no tools in py, or wrong tool name/ package name etc.

Reply:
Added test case to check the validation of package/tool name.